content/participants/ttjc@mritunjaysaha/index.md

```diff
@@ -28,18 +27,81 @@ I don't give up easily.
 
 ### help I need?
 
-It will be good for me if someone could help me learn the write practices for 
+It will be good for me if someone could help me learn the best practices for
 making a real world projects that would minimize bugs and increase performnace.
 
 ### how to contact me?
+
 - [Instagram](https://www.instagram.com/themritunjaysaha)
 - [Twitter](https://twitter.com/MritunjaySaha1)
 - [LinkedIn](https://www.linkedin.com/in/mritunjay-saha-89a04796)
 
 ### projects
 
-not a lot of projects are done. I will keep updating it with real links when done.
+#### Covid-19 Tracker
+
+_description_ A vanillaJs application to track the number of Covid-19 cases in India.
+
+_stack_ JavaScript
+
+_hosted link_ https://covidupdate.netlify.app/
+
+_github link_ https://github.com/mritunjaysaha/coronavirus-tracker
+
+#### AQI Monitor
+
+_description_ This allows us to see the air quality index of different cities at a time.
+
+_stack_ JavaScript
+
+_hosted link_ https://aqimonitor.netlify.app/
+
+_github link_ https://github.com/mritunjaysaha/air-quality-monitor
+
+#### Device Tracker
+
+_description_ Keeps track of the devices and shows their location on the Map
+
+_stack_ MongoDB is used for database, Node.js and express.js for backend, React for frontend and MapBox for Maps.
+
+_hosted link_ http://devtrkr.herokuapp.com/
+
+_github link_ https://github.com/mritunjaysaha/device-tracker
+
+#### Bluetooth Device Tracker
+
+_description_ Keeps track of the selected android devices within the bluetooth range
+
+_stack_ Java
+
+_github link_ https://github.com/mritunjaysaha/android-bt-v1
```

Comment:
Where is the hosted link!!
Please publish the app on playstore and provide the link.